rmw_cyclonedds/common.js

```diff
@@ -0,0 +1,2 @@
+import * as my_data from './data.js';
```

Comment:
I am a bit confused where the data is coming from. Aren't these supposed to be just json files?

Reply:
Data will come from github-action-benchmark tool which will be triggered by our other CI's. For example: https://github.com/DarkusAlphaHydranoid/moveit_middleware_benchmark_experimental/tree/gh-pages/rmw_fastrtps